src/http.cc

```diff
@@ -190,11 +190,13 @@ minio::http::Response minio::http::Request::execute() {
   if (debug) request.setOpt(new curlpp::Options::Verbose(true));
   if (ignore_cert_check) {
     request.setOpt(new curlpp::Options::SslVerifyPeer(false));
+    request.setOpt(new curlpp::Options::SslVerifyHost(0L));
   }
 
   if (url.https) {
     if (!ssl_cert_file.empty()) {
       request.setOpt(new curlpp::Options::SslVerifyPeer(true));
+      request.setOpt(new curlpp::Options::SslVerifyHost(2L));
```

Comment:
For passed TLS certificate, we do not need to set this option and let be how `libcurl` handle this.

Reply:
Fair, this might have been a little pedantic of me. I'm fine with removing it and allowing `libcurl`'s default behavior.

Reply:
It is removed.